src/App.js

```diff
@@ -24,10 +26,17 @@ const mapDispatchToProps = dispatch =>
       clearError: command.clearError.emit.create,
       updateCommand: command.update.emit.create,
       runCommand: command.run.emit.create,
+      deleteCommand: command.delete.emit.create,
     }, dispatch),
   });
 
 class App extends Component {
+  state = {
```

Comment:
let's to this a more generic and consistent way:
```
 state = {
    modalOpen: {
      confirmation: false,
    },
    modalData: {},
  };

  handleModalOpen = (modalType) => (modalData = {}) => {
    this.setState(({ modalOpen }) => ({ modalOpen: { ...modalOpen, [modalType]: true }, modalData }));
  };

  handleModalClose = (modalType) => () => {
    this.setState(({ modalOpen }) => ({ modalOpen: { ...modalOpen, [modalType]: false }, modalData: {} }));
  };
```

Reply:
this can be later extracted to a HOC 
the usage will be like this: `onClick={ this.handleModalOpen('<modalName>') }`

Reply:
also, notice the usage of functional `setState` in the code: we do this since we use the data from the current `state` (or `props`) to generate the new `state`